support/index.adoc

```diff
@@ -40,7 +40,9 @@ endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 
 You can xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-by-remote-health-monitoring[review telemetry information].
 
+ifndef::openshift-rosa,openshift-rosa-hcp[]
 If you have enabled remote health reporting, xref:../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#using-insights-to-identify-issues-with-your-cluster[Use Insights to identify issues with your cluster]. You can optionally disable remote health reporting.
+endif::openshift-rosa,openshift-rosa-hcp[]
 
 [id='support-overview-gather-data-cluster']
 == Gather data about your cluster
```

Comment:
For consistenceny with next bullet replace 
"*The must-gather tool*:" 
with 
"*must-gatherl*:"